catppuccin/__init__.py

```diff
@@ -1,4 +1,13 @@
 """🐍 Soothing pastel theme for Python."""
+
+import importlib.util
+
+from catppuccin.extras.matplotlib import _register_colormap_list, _register_styles
```

Comment:
this being unconditional is problematic as matplotlib is an optional dependency, so a clean install with no extras produces the following problem:

```python
>>> import catppuccin
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "X:\catppuccin\python\catppuccin\__init__.py", line 5, in <module>
    from catppuccin.extras.matplotlib import _register_colormap_list, _register_styles
  File "X:\catppuccin\python\catppuccin\extras\matplotlib.py", line 9, in <module>
    import matplotlib as mpl
ModuleNotFoundError: No module named 'matplotlib'
```

moving the imports into the conditional block below might be a good option.

Reply:
Ah, yes! Thanks for reviewing, I missed that. I'll do the change and add a bit of documentation about the `matplotlib` integration.